src/NuGet.Core/NuGet.Protocol.Core.v3/DependencyInfo/Utils.cs

```diff
@@ -20,7 +20,7 @@ internal static class Utils
         public static VersionRange CreateVersionRange(string stringToParse, bool includePrerelease)
```

Comment:
looks like includePrerelease is not used in CreateVersionRange, Why pass includePrerelease in?